cmd/get/releases/command.go

```diff
@@ -0,0 +1,85 @@
+package releases
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/giantswarm/kubectl-gs/pkg/middleware"
+	"github.com/giantswarm/kubectl-gs/pkg/middleware/renewtoken"
+)
+
+const (
+	name  = "releases [release-name]"
+	alias = "release"
+
+	shortDescription = "Display one or more releases"
+	longDescription  = `Display one or more releases`
+
+	examples = `  # List all releases
+  kubectl gs get releases
+
+  # Get one specific release by its name
+  kubectl gs get release 1.4.2`
```

Comment:
This actually requires the `v` prefix to work. Or can we actually make it work both with and without the `v` prefix?
```suggestion
  kubectl gs get release v1.4.2`
```

Reply:
Fixed